core/import_data/import_records_section_C.py

```diff
@@ -80,28 +81,21 @@ def parse_sheet(df):
 
         previous_year_prices_obj = None
         for report_details in REPORT_COLUMNS:
-            try:
-                previous_year_text = row[report_details["previous"]]
-                previous_year_price = (
-                    float(previous_year_text) if previous_year_text else None
-                )
-            # some price values are not decimals. skip them for now
-            except ValueError:
+            # get previous year price
+            previous_year_text = row[report_details["previous"]]
+            previous_year_price = get_decimal_from_excel_string(previous_year_text)
+            if previous_year_text and previous_year_price is None:
                 logger.warning(
                     f"⚠️ [row: {index_row + OFFSET}][year: {report_details['year']}] "
-                    "Price value is not a number for previous year"
-                )
-
-            try:
-                current_year_text = row[report_details["year"]]
-                current_year_price = (
-                    float(current_year_text) if current_year_text else None
+                    f"Incorrect price value for previous year {previous_year_text}"
```

Comment:
este necesar sa punem previous year avand in vedere ca punem anul curent in acelasi warning?

Reply:
cred ca e mai bine sa avem in warning si textul care a fost pus ca pret si pe care nu il acceptam. 
E mai usor sa inteleg din warning ce nu e bine la pretul respectiv daca vad si pretul fara sa mai fie nevoie sa merg in sheet sa vad ce si cum